app/views/expenses/index.html.haml

```diff
@@ -7,11 +7,28 @@
         %th{:scope => "col"}= I18n.t('expenses.index.name')
         %th{:scope => "col"}= I18n.t('expenses.index.price')
         %th{:scope => "col"}= I18n.t('expenses.index.paid_on')
-        %th
+        %th{:scope => "col"}= I18n.t('expenses.sort.action')
+        %th{:scope => "col"}
+          .dropdown
+            %button#dropdownMenuButton1.btn.btn-secondary.dropdown-toggle{"aria-expanded": "false", "data-bs-toggle": "dropdown", type: "button"}
+              = I18n.t('expenses.sort.by')
+            %ul.dropdown-menu{"aria-labelledby": "dropdownMenuButton1"}
+              %li
+                = sortable "name"
+              %li
+                = sortable "price"
+              %li
+                = link_to t('expenses.sort.oldest'), { sort: "paid_on", direction: "asc" }, { class: "btn" }
+              %li
+                = link_to t('expenses.sort.newest'), { sort: "paid_on", direction: "desc" }, { class: "btn" }
```

Comment:
Why not `sortable` here? :P

Reply:
I was using two different sort methods and i want your opinion which of theese is better :D